sites/all/themes/backdropapi/style.css

```diff
@@ -5,3 +5,18 @@ td.active {
 th {
   background-color: inherit !important;
 }
+
+h2 {
+  font-weight: 500;
+}
+code {
+  border: none;
+  background-color: #F7F7F7;
+  margin-bottom: 2em;
+}
+
+blockquote.d7diff {
```

Comment:
Hi @docwilmot, this looks good to me. Could we make this a bit more generic a class so we could use it in the future for Backdrop 1.x to 2.x changes? How about just "change" or "diff"?